target/linux/lantiq/base-files/sbin/dsl_notify.sh

```diff
@@ -1,71 +1,35 @@
 #!/bin/sh
 #
-# This script is called by dsl_cpe_control whenever there is a DSL event,
-# we only actually care about the DSL_INTERFACE_STATUS events as these
-# tell us the line has either come up or gone down.
-#
-# The rest of the code is basically the same at the atm hotplug code
-#
-
-[ "$DSL_NOTIFICATION_TYPE" = "DSL_INTERFACE_STATUS" ] || exit 0
-
-. /usr/share/libubox/jshn.sh
-. /lib/functions.sh
-. /lib/functions/leds.sh
-
-led_dsl_up() {
-	case "$(config_get led_dsl trigger)" in
-	"netdev")
-		led_set_attr $1 "trigger" "netdev"
-		led_set_attr $1 "device_name" "$(config_get led_dsl dev)"
-		led_set_attr $1 "mode" "$(config_get led_dsl mode)"
-		;;
-	*)
-		led_on $1
-		;;
-	esac
-}
-
-include /lib/network
-scan_interfaces
+# This script is called by dsl_cpe_control whenever there is a DSL event.
+
+debug=0
```

Comment:
I intentionally inserted this debug code to give even people who don't have access to the intel/lantiq dsl-api documentation the possibility to create their own notify scripts.

without this information it is hard to know which "events" can happen and so on.